Restructure Public Cloud specific information (#666)
SUSE/doc-susemanager: modules/installation/pages/public-cloud-registration.adoc

[[public-cloud-registration]]
= Registration in the Public Clouds

> Since {productname} is preinstalled in Public Cloud images, there is no installation step required. However, {productname} Server needs to be registered with {scc} in order to receive updates.

Reviewer: ```suggestion
Because {productname} is preinstalled in Public Cloud images, there is no installation step required.
However, {productname} Server needs to be registered with {scc} to receive updates.
```
